tests/soap/test_soap.py

```diff
@@ -63,3 +64,67 @@ def test_soap_vectors() -> None:
             f"compared to the control system "
             f"for r_cut: {r_c}."
         )
+
+
+def test_soap_centers() -> None:
+    """Test the consistency of SOAP calculations on specific centers.
+
+    This test verifies that the SOAP calculation yields similar
+    values as a control calculation at different r_cut using a specific
+    selection for the centers. The calculation of SOAP is influenced by the
+    architecture of the machine it's run on. As a result, the values of the
+    SOAP components might exhibit minor variations. To disregard these
+    differences, the function np.allclose() is employed.
+
+    Control file path:
+        - tests/systems/octahedron.xyz
+
+    Dynsight function tested:
+        - dynsight.soap.saponify_trajectory()
+            - soaplmax = 8
+            - soapnmax = 8
+
+    r_cuts checked:
+        - [1.75, 2.0, 2.15, 2.3, 2.45, 2.60, 2.75]
+    """
+    # Define input and output files
+    original_dir = Path(__file__).absolute().parent
+    input_file = original_dir / "../systems/octahedron.xyz"
+
+    # Define r_cuts
+    soap_r_cuts = [1.75, 2.0, 2.15, 2.3, 2.45, 2.60, 2.75]
+    check_file = np.load(original_dir / "../systems/SOAP.npz")
+
+    universe = MDAnalysis.Universe(input_file, dt=1)
+
+    # Run SOAP calculation for different r_cuts
+    for i, r_c in enumerate(soap_r_cuts):
+        test_soap = dynsight.soap.saponify_trajectory(
+            universe=universe,
+            soaprcut=r_c,
+            soaplmax=8,
+            soapnmax=8,
+            soap_respectpbc=False,
+            centers="id 2 or id 3",
```

Comment:
I would like to see a test that also shows that SOAP with ID 2 != ID 3 (for example, proving that this has an impact).

Reply:
This is not completed?

Reply:
Oh, i see, you did it through an assert - I would like to see it done by changing the argument `centers`.